src/app/models/trip.js

```diff
@@ -0,0 +1,7 @@
+import Backbone from 'backbone';
+
+const Trip = Backbone.Model.extend({
```

Comment:
Consider adding a [`urlRoot`](http://backbonejs.org/#Model-urlRoot) attribute which would allow the Trip to pull data from the API.